micr_XPCTzarr/sub-LADAF-2020-31/ses-01/micr/sub-LADAF-2020-31_ses-01_sample-brain_XPCT.json

```diff
@@ -0,0 +1,57 @@
+{
+        "InstitutionName": "European Synchrotron Radiation Facility",
+        "StationName": "BM05 EBS dipole wiggler 0.85T",
+        "BodyPart": "BRAIN",
+        "SampleEnvironment": "ex vivo",
+        "SampleFixation": "formalin",
+        "SampleEmbedding": "mixed agar gel at 70% ethanol",
+        "PixelSize": [
+                25.08,
+                25.08,
+                25.08
+        ],
+        "PixelSizeUnits": "um",
+        "ScanParameters": {
+                "XStep": "n/a",
+                "XStages": 1,
+                "YStep": "n/a",
+                "YStages": 1,
+                "ZStep ": "2.2 mm",
+                "ZStages ": "2x79",
+                "Projections": 9990,
+                "RefN": "n/a",
+                "DarkN": 400,
+                "RefOn": "n/a",
+                "ScanningMode ": "continuous",
+                "ExposureTime": "0.036 s",
+                "AccExposureTime ": "0.006 s",
+                "AccFramesCount": 6,
+                "PropDistance": "3475 mm",
+                "Filters": "Mo 0.1mm Al 2mm SiO2 bars 12*5mm diameter",
+                "DetAvgEnergy": "93 keV",
+                "ScanGeometry": "quarter-acquisition, one scan in half-acquisition plus one annular scan",
+                "ScanRange": "360 deg",
+                "SensorName": "sCMOS PCO edge 4.2 CLHS",
+                "SensorMode": "rolling shutter",
+                "SensorPixelSize": "6.5 um",
+                "Magnification": 0.24,
+                "XPixelNum": 2048,
+                "YPixelNum": 176,
+                "OpticsType": "dzoom optic from BM05 based on Hasselblad 120mm F/4 macro objective",
+                "Scintillator": "LuAG:Ce 2000 um",
+                "SurfDoseRate": "10.5 Gy/s",
+                "VoiDoseRate": "10.5 Gy/s",
+                "VoiIntegDose": "2.48 kGy",
+                "ScanTime": "7.88 min",
+                "SeriesTime": "22 h"
+        },
+        "ProcessingParameters": {
+                "RefApproach": "reference jar with 70% ethanol, single reference",
+                "VolumeX": 5965,
+                "VolumeY": 5965,
+                "VolumeZ": 6990,
+                "32to16BitsMin": -0.04,
+                "32to16BitsMax": 0.1,
+                "Jp2ComprRatio": 10
+        }
```

Comment:
@dstansby @Guillaume-Gaisne 
How will you split the metadata?

Reply:
For now, we have 5 different metadata sections: 
- Acquisition: the metadata from the H5 file produced by the scan
- Processing: the metadata associated with the processing of a scan
- Donor info: the metadata about the donor
- Sample info: the metadata about the sample (name, preparation protocol, etc...)
- Proposal info: the metadata about the proposal under which the scan is made

And we will add an other section about the metadata coming from the registration of a subvolume

Reply:
I would just throw most of this away and keep the metadata in this PR the metadata required by BIDS. For HiP-CT the way we store the metadata and what metadata we store is still being finalised...